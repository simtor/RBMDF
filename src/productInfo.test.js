const productInfo = require("./productInfo.js")
const products = require("./medicineInfo.js")

jest.mock("./medicineInfo.js")


test("PL001730050", ()=>{
    expect(productInfo("PL001730050")).toBe(true)
    expect(productInfo("PL001730050")).toMatchSnapshot();
})

test("PL001655013", ()=>{
    expect(productInfo("PL001655013")).toBe(true);
    expect(productInfo("PL001655013")).toMatchSnapshot();
})

test("PL136060206", ()=>{
    expect(productInfo("PL136060206")).toBe(true);
    expect(productInfo("PL136060206")).toMatchSnapshot();
})


test("PL252980158", ()=>{
    expect(productInfo("PL252980158")).toBe(true);
    expect(productInfo("PL252980158")).toMatchSnapshot();
})

test("PL201170079", ()=>{
    expect(productInfo("PL201170079")).toBe(true);
    expect(productInfo("PL201170079")).toMatchSnapshot();
})
