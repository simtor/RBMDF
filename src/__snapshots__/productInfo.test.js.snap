// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PL001655013 1`] = `true`;

exports[`PL001730050 1`] = `true`;

exports[`PL136060206 1`] = `true`;

exports[`PL201170079 1`] = `true`;

exports[`PL252980158 1`] = `true`;
